app/validators/question_answer_comment_validator.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class QuestionAnswerCommentValidator < ActiveModel::Validator
+  def validate(record)
+    post_body_validator(record)
```

Comment:
Shouldn't this be called `comment_body_validator` or something?

Reply:
Oo good catch :+1: